airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -92,6 +93,15 @@ def _get_refresh_access_token_response(self):
         except requests.exceptions.RequestException as e:
             if e.response.status_code == 429 or e.response.status_code >= 500:
                 raise DefaultBackoffException(request=e.response.request, response=e.response)
+
+            error_content = e.response.json()
+            if e.response.status_code == 400 and error_content.get("error") == "invalid_grant":
```

Comment:
I think we should move the error codes, keys and values to class level so that they could be overridden in the connectors. If they are not, we should the default behavior should not be changed